spec/commands/csv_data_upload_spec.rb

```diff
@@ -194,19 +268,54 @@
     expect(assessment.questions[1].answer).to eq('Yes')
   end
 
+  it 'import CSV file with Geographies data' do
+    # USA is already created
+    expect_data_upload_results(
+      Geography,
+      geographies_csv,
+      new_records: 1, not_changed_records: 0, rows: 2, updated_records: 1
+    )
+    # subsequent import should not create or update any record
+    expect_data_upload_results(
+      Geography,
+      geographies_csv,
+      new_records: 0, not_changed_records: 2, rows: 2, updated_records: 0
+    )
+
+    geography = Geography.find_by(iso: 'USA')
+
+    expect(geography.legislative_process).to eq('Legislative process USA')
+    expect(geography.name).to eq('United States of America')
+    expect(geography.region).to eq('North America')
+    expect(geography.geography_type).to eq('country')
+    expect(geography.visibility_status).to eq('draft')
+    expect(geography.political_groups.size).to eq(2)
+    expect(geography.political_groups_list).to include('OECD', 'G20')
+    expect(geography.federal).to be_truthy
+    expect(geography.federal_details).to be
+  end
+
   def expect_data_upload_results(uploaded_resource_klass, csv, expected_details)
     uploader_name = uploaded_resource_klass.name.tr('::', '').pluralize
     command = Command::CsvDataUpload.new(uploader: uploader_name, file: csv)
 
     expect do
+      expect(command).to be_valid
       expect(command.call).to eq(true)
       expect(command.details.symbolize_keys).to eq(expected_details)
     end.to change(uploaded_resource_klass, :count).by(expected_details[:new_records])
   end
 
-  def fixture_file(filename)
+  def fixture_file(filename, content: nil)
+    file_path = "#{Rails.root}/spec/support/fixtures/files/#{filename}"
+
+    if content.present?
+      file_path = "#{Rails.root}/tmp/litigation_sides.csv"
```

Comment:
hmm, why we have reference to `litigation_sides` in generic `fixture_file` helper?:)

Reply:
oh shit, good catch. That file name for tmp file does not actually matter but should be `tmp/#{filename}`